src/components/protected-route/ProtectedRoute.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+export default function ProtectedRoute({
+  component: Component,
```

Comment:
i would move this kind of components in folder HOC (may be inside "components/hoc")

Reply:
moved